mobile_app/README.md

```diff
@@ -29,3 +29,19 @@ separate build machine.
 The expectation is that the branch name will be some unique identifier like the
 jenkins job number and the date.
 	
+
+checkout_build_repos
+--------------------
+
+Checks out out a code and config repository and sets them up for building, by creating a properties file to point the code at the config.
```

Comment:
"Checks *out out* a code and config repository" -> "Checks out code and config repositories"